### PR TITLE
Feature/2709 update native sdks

### PR DESCRIPTION
**Pull request by nicolas-chaix-didomi**
Closes CMP-2709 and CMP-2716

- Update native SDKs
- Add missing iOS implementation for `getQueryStringForWebView`
- Add new parameter `regulation` for `UserStatus` model
- Update kotlin version and gradle wrapper

### CODE REVIEW

File: android/src/main/kotlin/io/didomi/fluttersdk/DidomiPlugin.kt [hunk 1]
```diff
@@ -294,7 +297,7 @@ class DidomiPlugin : FlutterPlugin, MethodCallHandler, ActivityAware {
      */
     private fun getJavaScriptForWebView(result: Result) {
         try {
-            val text = Didomi.getInstance().javaScriptForWebView
+            val text = Didomi.getInstance().getJavaScriptForWebView()
```
Comment (pmerlet-at-didomi) [inline android/src/main/kotlin/io/didomi/fluttersdk/DidomiPlugin.kt:300]: Why is this change necessary?

Comment (nicolas-chaix-didomi) [reply android/src/main/kotlin/io/didomi/fluttersdk/DidomiPlugin.kt:300]: During Didomi class migration to Kotlin, we merged `javaScriptForWebView` getter and `getJavaScriptForWebView(extra:String)` to `getJavaScriptForWebView(extra: String?)`
